UOP1_Project/Assets/Scripts/Inventory/Item.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+
+// Created with collaboration from:
+// https://forum.unity.com/threads/inventory-system.980646/
+public abstract class Item : ScriptableObject
+{
+
+	// TODO: review if this enum redundant due to abstraction 
+	enum  ItemType
+	{
+		CharacterCustomisation,
+		Ingredients,
+		Utensils,
+		Dishes,
+	}
```

Comment:
I agree with the comment, I think this is redundant - ideally we'll rely on inheritance to sort this out. For example, "Ingredient" already inherits Item.